### ISSUE TITLE
Outputs are not entirely random given seed

### ISSUE THREAD

**Issue by nemosmithasf** [zeh/prando#16]
Following up on #1. I know the issue is marked as resolved, but it seems like the output is still not truly random given incrementing integer seeds.

For example, I am instantiating multiple Prando instances, and using "between" to pick random points on a map (each pick is using a unique Prando instance). The seed is incrementing integers (1, 2, 3...) for each subsequent instance.

The result is 

The points circled are the result of adjacent integers. As you can see, there's clearly a non-random pattern when seeds are adjacent integers.

**Comment by hMihaiDavid** [comment]
Try using a long random string as a seed.

**Comment by zeh** [comment]
This is a bit of a known issue with this sort of generator. The numbers are "random enough" for an instance of RNG, but if you start an instance with a seed that is somewhat similar to a different instance, the random numbers will also be somewhat similar at the beginning.

It's recommended to use a "more random" seed to start, or at least a larger one. For example, instead of seed `1, 2, 3` and `Prando(seed)`, you could do something like `Prando(seed * 123456)` or some such. Doing `rng.skip(5)` or some such at the start also helps shuffle the numbers a bit.

TL;DR not ideal, but it's expected.
